scrapers/koji_scraper.py

```diff
@@ -88,6 +100,21 @@ def update_neo4j(self, builds):
             build.owner.connect(user)
             user.koji_builds.connect(build)
 
+            try:
+                # Getting tag related to the current build
+                tag_dict = self.get_build_tag(build_dict['id'])[0]
+            except IndexError:
+                # Continue if no active tag is found for the current build
+                continue
+
+            tag = KojiTag.create_or_update(dict(
```

Comment:
Can you change this to `get_or_update` since you'll never need to update this node?